python/systematics.py

```diff
@@ -1,17 +1,26 @@
 import CombineHarvester.CombineTools.ch as ch
 
+
+bbhsignals = ['bbH_htt','bbH_hww','intH_htt','intH_hww']
+hwwprocs_sig = ['bbH_hww','ggH_bb_hww','intH_hww']
+httprocs_sig = ['bbH_htt','ggH_bb_htt','intH_htt']
+
+h125ttprocs = ['ggH_htt','qqH_htt','WH_htt','ZH_htt','ttH_htt']
+h125wwprocs = ['ggH_hww','qqH_hww','ttH_hww','ZH_hww','WH_hww']
```

Comment:
I think we need the old process names here, see comment here: https://github.com/raspereza/bbHRun2Legacy/pull/10/files#r1333247905
(unless I missed something about what is happening)